src/db/OrganizationTypes.ts

```diff
@@ -0,0 +1,120 @@
+import mongoose from 'mongoose'
+import { MUUID } from 'uuid-mongodb'
+
+import { ChangeRecordMetadataType } from './ChangeLogType.js'
+
+/**
+ * Organizations are OpenBeta accounts that are owned by organizations
+ * instead of people. For instance, the Boulder Climbing Community might own
+ * one such account. Organizations may have one or more 'org_admins' who
+ * manage the account. They may update organizations' profiles and perform
+ * special actions under the auspices of the organization. For example, org_admins
+ * of local climbing organizations (LCO) may put up bird nesting notices on
+ * the area pages associated with the LCO.
+ * */
+export type OrganizationType = {
```

Comment:
Actually not sure when to make this a type vs interface. What's the difference?

Reply:
I think it's a matter of style. I'm indifferent.